packages/portfolio/src/ui/portfolio-ui-tokens.tsx

```diff
@@ -6,9 +6,9 @@ import { ellipsify } from '@workspace/ui/lib/ellipsify'
 
 import { PortfolioUiExplorerLink } from './portfolio-ui-explorer-link.tsx'
 
-export function AccountUiTokens({ network, items }: { network: Network; items: GetTokenAccountsResult }) {
+export function WalletUiTokens({ network, items }: { network: Network; items: GetTokenAccountsResult }) {
```

Comment:
It looks like biome doesn't sort the properties here as it passes `lint` and this is not sorted. 

Created issue #365